webscraper.js

```diff
@@ -35,12 +39,42 @@ if (require.main === module) {
                                             };
         }
 
-        return doc_dimensions;
-      }).then(function(dimensions) {
+        setTimeout(function () {
+           callback({
+              html: src,
+              css: node_properties,
+              folder : dir
+           });
+        });
+
+      }, item).then(function(doc_info) {
```

Comment:
Is `item` used? As I understand it from the Selenium docs, it gets passed into the function (`function(dir)`, which, btw, you should name; always name nontrivial functions, as a sort of comment), but I don't think it's then used within.